FEEDBACK.md

```diff
@@ -0,0 +1,9 @@
+General comments/feedback:
+
+Overall think it looks great. Clear and easy to follow, and plenty of good content. Here's some general feedback. Let me know if any of these sound good and I can make separate PRs for them:
+
+1. Should we include a brief overview of programming in general at the very beginning? High level, ‘what is a program?’ sort of stuff?
+2.  It might be helpful to add an example at the beginning of conditionals that makes the difference between a true and false condition really obvious. For example we could use the window size as the conditional and manually resize the window to show different outcomes
+3. For the rock, paper, scissors, game in practice problems what if we set it up so two people could actually play together? We could setup two window.prompt variables and an empty function and have the students complete the rest from there
+4. Rory mentioned that he generally teaches loops before arrays because you can setup a problem where you need to loop over/sort through a bunch of variables, and then present arrays as the solution. Also that teaching ‘while’ loops, specifically with an explicit iterator, can be a nice transition to ‘for’ loops (but I think you mentioned you didn’t want to include ‘while’ loops in this workshop?). But ultimately he said it’s pretty subjective and could really go either way
+5. Would it be helpful if I create some more practice problems and/or activity problems? Or do you think we’re good in that regard? 
```

Comment:
Cool yeah definitely agree we need to get moving on the syllabus (on that note I've actually started creating some content for the Git/Github section of Unit 1). If I can think of some problems/challenges that would be quick to write up and add to the workshop I'll make some PRs, otherwise will focus on the syllabus